content/chainguard/libraries/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Chainguard Libraries FAQ"
+linktitle: "FAQ"
+type: "article"
+description: "Frequently Asked Questions and answers for Chainguard Libraries users"
```

Comment:
I would make all title case or all lower case except for proper nouns just to be consistent

Reply:
Agreed .. switches to sentence case to improve readability.